config/secrets.yml

```diff
@@ -14,16 +14,18 @@ development:
   secret_key_base: 36746f0eb3c57b16be0278bdf9a9418ec225ecd5f9a3183697a3dc0dfd3aa1d05d285b3ed42b77c92ae6deee7af6a2b9bc87f119eca9c74810dcc43ddd5eb02b
   google_api_key: <%= ENV["GOOGLE_API_KEY"] %>
   rate_limit_token: pfB6uNKYC8sB9PVgBLdwFToN
+  govuk_basic_auth_credentials: Basic c81a772d7703bfdf8cfc79a0
```

Comment:
This should not have the `Basic` anymore.

Reply:
How do we feel about adding be the `Basic` at initialization? If I was getting a string from a hash called `hosts_with_basic_authorization` I might expect it to have that already.

Reply:
I would expect a variable called `hosts_with_authorization` to include the `Basic` as it could be any type of authorisation, but since we know they can only be basic authorisation I would expect the strings not to include that.